src/index.html

```diff
@@ -5,8 +5,347 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <picture class="header__logo">
+        <a href="#home"><img src="./images/logo.png" alt="mate-logo"></a>
+      </picture>
+
+      <nav>
+        <ul class="header__ul">
+          <li class="header__li"><a class="is-active header__a" href="#">apple</a></li>
+          <li class="header__li"><a class="header__a" href="#">samsung</a></li>
```

Comment:
Need to fix it everywhere
```suggestion
          <li class="header__li">
            <a class="header__a" href="#">samsung</a>
          </li>
```